examples/vm_os_upgrade/05_clone_from_source.yml

```diff
@@ -0,0 +1,45 @@
+- name: Create snapshot for source VM
+  scale_computing.hypercore.vm_snapshot:
+    state: present
+    vm_name: "{{ source_vm_name }}"
+    label: "{{ source_snapshot_label }}"
+  register: vm_created_snapshot
+
+- name: Clone source VM
+  scale_computing.hypercore.vm_clone:
+    vm_name: "{{ cloned_vm_name }}"
+    source_vm_name: "{{ source_vm_name }}"  # latest snapshot will be cloned
```

Comment:
Just add here `source_snapshot_label:` to use the specific snapshot. Otherwise I think a new snapshot is taken, and we get VM cloned from that one.

Reply:
Thanks for comment, added!